packages/sources/coinpaprika/src/util.ts

```diff
@@ -0,0 +1,35 @@
+import { makeExecute } from './adapter'
+import { executeSync } from '@chainlink/ea-bootstrap'
+import { CoinsResponse } from './endpoint/coins'
+
+export const getCoinIds = (id: string): Promise<CoinsResponse> => {
+  return new Promise((resolve, reject) => {
+    const execute = makeExecute()
+    const executeWithMiddleware = executeSync(execute)
+    const options = {
+      data: {
+        endpoint: 'coins',
+      },
+      method: 'post',
+      id,
+    }
+    console.log(options)
```

Comment:
Should we be using our logger here?

Reply:
It will get logged by Requester. Removing this dev console.log.